docker/drama-free-django/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM centos:7
```

Comment:
The current versions of the build and test scripts use `centos:6`(also documented [here](https://github.com/cfpb/cfgov-refresh/tree/master/docker/drama-free-django)). I did this deliberately to try to match the current setup. Is there a particular reason to change that version here?

Reply:
That was not intentional.  Will fix.